.codeclimate.yml

```diff
@@ -1,6 +1,10 @@
 engines:
   fixme:
-    enabled: false
+    enabled: true
+    config:
+      strings:
+      - legacy
+      - WIP
```

Comment:
We don't want to look for any of the defaults in `fixme` engine because of false positives (hence it being disabled);

These might not be the best strings, but I thought they had potential and helped demonstrate to users how to use custom strings.